.travis.yml

```diff
@@ -12,6 +12,7 @@ before_install:
 
 script:
   - ./.ci/build.sh
+  - rm -rf private/
```

Comment:
And this one as well...

Reply:
and in case somehow the directory is still existing here, it should be removed.  Extra careful.